src/database/Database.cpp

```diff
@@ -108,41 +108,14 @@ Database::applySchemaUpgrade(unsigned long vers)
 {
     clearPreparedStatementCache();
 
-    switch (vers)
+    if (vers <= 6)
```

Comment:
keep the switch as it makes the code easier to modify in the future (we'll continue to follow the same pattern)